### PR TITLE
kotlin: update to 1.1

### PR DESCRIPTION
**Pull request by breun**
https://blog.jetbrains.com/kotlin/2017/03/kotlin-1-1/
https://github.com/JetBrains/kotlin/releases/tag/v1.1

### CODE REVIEW

**Comment by l2dy** [comment]
Please don't add links to release notes in the commit message.

**Comment by breun** [comment]
I've been doing this ever since I started updating ports. I didn't know this was not okay and also don't see https://trac.macports.org/wiki/CommitMessages saying anything about this. May I ask why this is a problem? Personally I enjoy such references to more information.

**Comment by l2dy** [comment]
Nobody else was doing that so I assume this be a convention. The extra information isn't useful unless there's some API breaking changes that could break dependent ports.

**Comment by g5pw** [comment]
Well, it's extra information, and it can't hurt in an update commit (since there's not really much to say)

File: lang/kotlin/Portfile [hunk 1]
```diff
@@ -19,10 +19,10 @@ long_description    Kotlin is a pragmatic programming language for JVM \
                     and is focused on interoperability, safety, clarity \
                     and tooling support.
 
-homepage            http://kotlinlang.org/
+homepage            https://kotlinlang.org/
```
Comment (g5pw) [inline lang/kotlin/Portfile]: This is *very* good. We should probably check for https versions of all our URLs.

**Pull request merged**